types/gapi.client.calendar/index.d.ts

```diff
@@ -11,7 +11,7 @@
 // This file was generated by https://github.com/Maxim-Mazurok/google-api-typings-generator. Please do not edit it manually.
 // In case of any problems please post issue to https://github.com/Maxim-Mazurok/google-api-typings-generator
 // Generated from: https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest
-// Revision: 20200820
+// Revision: 20200822
```

Comment:
should we avoid revision updates if nothing changed? :)

Reply:
Yes, definitely, and we already had it. Seems like a regression, I'll fix that, sorry for the trouble!

Reply:
Haha no worries!